packages/abi-typegen/test/fixtures/templates/contract/dts.hbs

```diff
@@ -109,8 +109,8 @@ export class MyContractAbi extends Contract {
     types_b512: InvokeFunction<[x: string], string>;
     types_bool: InvokeFunction<[x: boolean], boolean>;
     types_bytes: InvokeFunction<[x: Bytes], Bytes>;
-    types_empty: InvokeFunction<[], void>;
-    types_empty_then_value: InvokeFunction<[y: BigNumberish], void>;
```

Comment:
I think `void` as a return is nicer than `undefined` and it's implied in the mind of a developer that a `void` return is `undefined` anyways.

Reply:
b3d704821f90640105688c2888259693938bfbf0